validator/build.gradle

```diff
@@ -21,13 +21,23 @@ dependencies {
     implementation "commons-io:commons-io:2.4" // not sure why this isn't getting picked up from hri-flink-pipeline-core
     implementation "org.apache.flink:flink-connector-kafka_${scalaBinaryVersion}:${flinkVersion}" // not sure why this isn't getting picked up from hri-flink-pipeline-core
     implementation "com.ibm.fhir:fhir-model:4.7.1"
+    compileOnly("org.scala-lang:scala-compiler") {
```

Comment:
I don't see the flink-scala stuff pulled in as implementation, so won't need this unless one of those other dependencies is pulling it in.